### ISSUE TITLE
Circumferential skin creases on extremities

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7737]
**Preferred term label:**
Circumferential skin creases on extremities

**Synonyms**

**Definition (free text, please give PubMed ID)**
Ringed creases of the skin of the extremities related to folding of excess skin.

**Parent term (use hpo.jax.org/app)**
Localized skin lesion HP:0011355

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
616734 
PMID: 26637975  4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
